NEWS.md

```diff
@@ -1,5 +1,9 @@
 # teal 0.12.0.9011
 
+### New features
+
+* Added the `validate_inputs` function that produces informative error messages in app output.
```

Comment:
> I may also move this below the "Major breaking changes" section but since we're going to tidy up the NEWS before the next release i think it's fine

Is it not a "new feature"?

Reply:
it is yes - I would move the whole New feature section below the "we have changed how teal modules work after 5 years" breaking change mention. But this can all be done later